packages/snack-sdk/src/utils.ts

```diff
@@ -62,3 +62,15 @@ export function createError(config: {
   if (config.stack) error.stack = config.stack;
   return error;
 }
+
+export function createUserHeader(user?: SnackUser): { [key: string]: string } {
+  if (user?.sessionSecret) {
+    return { 'Expo-Session': user.sessionSecret };
+  }
+
+  if (user?.accessToken) {
+    return { Authorization: `bearer ${user.accessToken}` };
```

Comment:
Does it make any difference how `Bearer` is capitalized?

```suggestion
    return { Authorization: `Bearer ${user.accessToken}` };
```

Reply:
You are right, it should be capitalized, afaik bearer tokens originates from OAuth. [In that spec](https://tools.ietf.org/html/rfc6750#section-2.1) it's defined with a capital. We also [use the capitalized one in CLI](https://github.com/expo/expo-cli/blob/master/packages/xdl/src/ApiV2.ts#L205)